library/test/config/setup.ts

```diff
@@ -0,0 +1,20 @@
+import path from 'path'
+import puppeteer, { Browser } from 'puppeteer'
+import { getExtensionId } from '../utils/extension'
+
+const EXTENSION_PATH = path.join(__dirname, '..', '..', '..', 'dist')
+
+export default async () => {
+  const browser: Browser = await puppeteer.launch({
+    headless: false,
+    args: [
+      `--disable-extensions-except=${EXTENSION_PATH}`,
+      `--load-extension=${EXTENSION_PATH}`,
```

Comment:
shouldn't the `swarm-extension` also be added?

Reply:
Yes, that's correct. I have added the Swarm extension here as well. But now when I tried to compile the Swarm extension I got an error. But that's not related to this PR. It should be investigated separately.

Reply:
I have fixed that issue in the login branch.